### PR TITLE
Feature: Display selected federal state in texts for mask state (EXPOSUREAPP-13911)

### PR DESCRIPTION
**Pull request by mlenkeit**

### CODE REVIEW

File: resources/i18n/de/strings.xml [hunk 1]
```diff
@@ -170,17 +170,82 @@
...
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - HB -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_HB">"Bremen\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - HH -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_HH">"Hamburg\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - HE -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_HE">"Hessen\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - MV -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_MV">"Mecklenburg-Vorpommern\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - NI -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_NI">"Niedersachsen\nEine Maske ist dennoch empfohlen"</string>
```
Comment (Ein-Tim) [inline resources/i18n/de/strings.xml:234]: Why is the word "dennoch" in here? Is there a paragraph before which says something like "Von der Maskenpflicht ausgenommen"?

Comment (mlenkeit) [reply resources/i18n/de/strings.xml]: This particular text sequence was dropped already 😅 You find the latest proposal in https://github.com/corona-warn-app/cwa-app-ccl/pull/106/files#diff-3526f651c4d797d161de635d89eff2562be6a1ed70ca0ad709556114c9c2164c. The _dennoch_ is still there, but it's embedded into a sentence.

Comment (Ein-Tim) [reply resources/i18n/de/strings.xml]: Fair enough, thanks!

File: resources/i18n/de/strings.xml [hunk 2]
```diff
@@ -170,20 +170,16 @@
...
   <!-- XHED: Mask State - MASK_REQUIRED Card Title -->
   <string name="MASK_STATE_MASK_REQUIRED_TITLE_TEXT">"Maskenpflicht"</string>
-  <!-- XHED: Mask State - MASK_REQUIRED Card Subtitle -->
-  <string name="MASK_STATE_MASK_REQUIRED_SUBTITLE_TEXT">"Sie sind nicht von der Maskenpflicht ausgenommen"</string>
-  <!-- XTXT: Mask State - MASK_REQUIRED Card Body -->
-  <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT">"Von der Maskenpflicht sind alle Personen befreit, die innerhalb der letzten 3 Monate geimpft wurden oder genesen sind oder innerhalb der letzten 24 Stunden negativ getestet wurden."</string>
-  <!-- XTXT: Mask State - MASK_REQUIRED Badge Text -->
-  <string name="MASK_STATE_MASK_REQUIRED_BADGE_TEXT">"Maskenpflicht"</string>
+  <!-- XTXT: Mask State - MASK_REQUIRED Card Body - Federal default rules -->
+  <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT_BASIC">"Sie sind nicht von der Maskenpflicht ausgenommen.\n\nVon der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen, die innerhalb der letzten 24 Stunden negativ getestet wurden."</string>
```
Comment (MikeMcC399) [inline resources/i18n/de/strings.xml:174]: The text is repeated:

"Von der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen"

File: test/fixtures/ccl/dcc-series-wallet-info-mask-state.yaml [hunk 3]
```diff
@@ -709,6 +850,154 @@
...
\ No newline at end of file
+            badgeText: Maskenpflicht
+    # federal default rules
+    - time: rat1+P5D # some long duration
+      scenarioIdentifier: ''
+      assertions:
+        walletInfo:
+          maskState:
+            longText:
+              de: 'Sie sind nicht von der Maskenpflicht ausgenommen.\n\nVon der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen, die innerhalb der letzten 24 Stunden negativ getestet wurden.'
```
Comment (MikeMcC399) [inline test/fixtures/ccl/dcc-series-wallet-info-mask-state.yaml:861]: Also repeated here:

"Von der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen"

Comment (mlenkeit) [reply test/fixtures/ccl/dcc-series-wallet-info-mask-state.yaml]: Thanks @MikeMcC399 for spotting this, fixed with https://github.com/corona-warn-app/cwa-app-ccl/pull/106/commits/da8f1ab3bc1832f6987fc5abd9108d9cc5f98ac0

File: resources/i18n/de/strings.xml (hunk 2 above)
Comment (mlenkeit) [reply resources/i18n/de/strings.xml]: Thanks @MikeMcC399 for spotting this, fixed with https://github.com/corona-warn-app/cwa-app-ccl/pull/106/commits/da8f1ab3bc1832f6987fc5abd9108d9cc5f98ac0

**Review by KathrinLuecke — COMMENTED** [review]
UA approved

**Pull request merged**